templates/github/.github/workflows/scripts/before_script.sh.j2

```diff
@@ -29,6 +29,11 @@ if [[ "$TEST" == 'pulp' || "$TEST" == 'performance' || "$TEST" == 'upgrade' || "
   cmd_prefix dnf install -yq lsof which dnf-plugins-core
 fi
 
+if [[ "$REDIS_DISABLED" == true ]]; then
```

Comment:
This fails here:
https://github.com/pulp/pulpcore/runs/4319788337?check_suite_focus=true#step:12:143

```suggestion
if [[ "${REDIS_DISABLED:-false}" == true ]]; then
```